packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/dropDownEditor.tests.js

```diff
@@ -457,6 +457,66 @@ QUnit.module('focus policy', () => {
         assert.ok($dropDownEditor.hasClass('dx-state-focused'), 'Widget is focused after click on clearButton');
     });
 
+    QUnit.test('editor should fire onFocusIn once, if fieldTemplate is used', function(assert) {
+        const onFocusInHandler = sinon.spy();
+
+        const $dropDownEditor = $('#dropDownEditorLazy').dxDropDownEditor({
+            items: [
+                { name: 'one', id: 1 },
+                { name: 'two', id: 2 },
+                { name: 'three', id: 3 },
+            ],
+            displayExpr: 'name',
+            valueExpr: 'id',
+            value: 1,
```

Comment:
Cannot we minimize the test? Now a lot of properties looks excess

Reply:
[7894ae7f11962bb566873551ef637e5348c23f52]